db/rlang-duktape

```diff
@@ -4,7 +4,7 @@ R2PM_GIT "https://github.com/radareorg/radare2-rlang"
 R2PM_DESC "[rlang] Duktape JS RLang plugin #!duktape"
 
 R2PM_INSTALL() {
-	${MAKE} clean || exit 1
+	./configure || exit 1
```

Comment:
can you just add this line after the `make clean` and also change `${MAKE} clean || exit 1` to `${MAKE} clean || sleep 0` so if it fails, we do not care.